### PR TITLE
Work details popup

### PR DESCRIPTION
**Pull request by RWUBAKWANAYO**
# In this project we:

- Add popup section in HTML file
- Add styles of popup elements
- Change the view of popup on both Desktop and mobile
- Create  objects inside the array to hold work details data
- Change popup content dynamically according to the targeted element
- fix linter errors in HTML, CSS, javaScript file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -152,7 +152,47 @@ <h2>Multi-Post Stories</h2>
...
+                <a href="#" target="_blank" class="popup-link-cont" id="sourceCode">
+                  <span class="popup-link">
+                <span>See Source</span>
+                  <i class="fa-brands fa-github"></i>
+                  </span>
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
```
Comment (sja-thedude) [inline index.html]: - [ ] Kindly add the entire Projects section dynamically. Kindly make changes in existing code by removing already created HTML code for the projects list, Instead, it should be added via JavaScript dynamically.

File: style.css [hunk 2]
```diff
@@ -353,6 +348,137 @@ header {
...
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 21;
+  background-color: rgba(64, 83, 252, 0.24);
+  backdrop-filter: blur(4px);
+  transition: 0.3s ease-in-out;
+}
+
```
Comment (sja-thedude) [inline style.css:363]: - [ ] Kindly make sure to add width on right and left sides of the image. As you can see the image is only having a height at the top and bottom and according to the Figma design, it should be in the centre.
- [ ] Kindly make the corners of the image container in the popup window curve at the end as per the Figma design.
- Figma design
Screenshot 2022-04-06 at 2 02 21 AM
Your design:
Screenshot 2022-04-06 at 2 04 53 AM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @RWUBAKWANAYO, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Linters set up properly
✔️ Good PR Description and Title
✔️ Good 👏 job in working on the popup window

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (RWUBAKWANAYO) [reply style.css:363]: Hi @sja-thedude,
I appreciate the feedback you provide on this project and I want to ask the following things:

- Are you suggesting that every content of the work section has to be dynamic?
- I don't understand very well what you're trying to mean by "make the corners of the image container in the popup window curve at the end as per the Figma design"

Comment (sja-thedude) [reply style.css:363]: Hi @RWUBAKWANAYO, 
Yes I am suggesting you to make the entire work section dynamic.

I try to mean that as you can see in the images below that the figma design have border radius which makes it rounded corners so you have to add border radius to the image which is on the popup window.

Your design:
image

Figma design:
image

**Review by phelian23 — APPROVED** [review]
Hi @RWUBAKWANAYO, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by RWUBAKWANAYO** [comment]
merge changes from the Work-Details-Popup branch to the main branch

**Pull request merged**